app/views/articles/index.html.erb

```diff
@@ -6,4 +6,4 @@
     <% end %>
   </ul>
 </div>
-<a href="/articles/new">Make a new article</a>
\ No newline at end of file
+<%= link_to "Make a new article", new_article_path, class: "new_article" %>
```

Comment:
Is this class a CSS selector? Asking because it isn't in the same form CSS classes usually are. If it is, could you call it to determine a within block in a test? Thanks! @carmer
@s-espinosa